oasislmf/utils/data.py

```diff
@@ -293,7 +295,13 @@ def get_dataframe(
     na_values = list(PANDAS_DEFAULT_NULL_VALUES.difference(['NA']))
 
     if src_fp and src_type == 'csv':
-        df = pd.read_csv(src_fp, float_precision=float_precision, memory_map=memory_map, keep_default_na=False, na_values=na_values)
+        # Find flexible fields in loc file and set their data types to that of
+        # FlexiLocZZZ
+        if 'FlexiLocZZZ' in col_dtypes.keys():
```

Comment:
Hard-coding column names isn't recommended - it would be better to add this column to the loc. JSON profile (in static data), as it is a loc. file column, describe it as part of the OED hierarchy (see how acc. num., loc. num., etc. are defined in the loc. profile), and use the OED hierarchy terms wrapper (in `utils/profiles.py` I think) to access the column name internally. This probably sounds very complicated, but it is about the principle of making the code independent of column names of files where the schema is subject to change externally.

For the moment, you can keep it - perhaps you can do this in the next release.